api/v1beta2/ibmpowervsmachine_types.go

```diff
@@ -121,6 +131,20 @@ type IBMPowerVSMachineSpec struct {
 	// ProviderID is the unique identifier as specified by the cloud provider.
 	// +optional
 	ProviderID *string `json:"providerID,omitempty"`
+
+	// Ignition defined options related to the bootstrapping systems where Ignition is used.
+	// +optional
+	Ignition *Ignition `json:"ignition,omitempty"`
```

Comment:
why a struct here? right now I see just a version, do we expect anything else in the future?